doc/changes.md

```diff
@@ -1,5 +1,118 @@
 # Changelog
 
+## Release v0.11.0 (2024/02/01)
+
+[![Digital Object Identifier for PyGMT v0.11.0](https://zenodo.org/badge/DOI/10.5281/zenodo.10578540.svg)](https://doi.org/10.5281/zenodo.10578540)
+
+### Highlights
+
+* Tutorial for table inputs ([#2722](https://github.com/GenericMappingTools/pygmt/pull/2722)) and gallery example for choropleth map ([#2796](https://github.com/GenericMappingTools/pygmt/pull/2796))
+* Easy access to remote datasets of planetary reliefs (Moon, Venus, Mars, Mercury, Pluto) ([#2674](https://github.com/GenericMappingTools/pygmt/pull/2674), [#2906](https://github.com/GenericMappingTools/pygmt/pull/2906), [#2847](https://github.com/GenericMappingTools/pygmt/pull/2847))
```

Comment:
```suggestion
* Easy access to remote datasets of planetary reliefs (Moon, Venus, Mars, Mercury, Pluto) ([#2674](https://github.com/GenericMappingTools/pygmt/pull/2674), [#2906](https://github.com/GenericMappingTools/pygmt/pull/2906), [#2847](https://github.com/GenericMappingTools/pygmt/pull/2847), [#3027](https://github.com/GenericMappingTools/pygmt/pull/3027), [#3028](https://github.com/GenericMappingTools/pygmt/pull/3028))
```

Reply:
Did you want the Mercury and Pluto datasets in v0.11.0? Or push it to the next release?

Reply:
Prefer to have them in v0.11.0 so that all new platenary datasets are available via PyGMT.